File: ScrumChores.Model/Entities/Sprint.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace ScrumChores.Model.Entities
{
    public class Sprint
    {
        [Key]
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        public Guid SprintID { get; set; }
        public string SprintName { get; set; }
        public DateTime SprintStartDate { get; set; }
        public DateTime SprintEndDate { get; set; }
        public virtual List<Story> Stories { get; set; } 
    }
}

File: ScrumChoresPublicKO/Scripts/ScrumChores/Common/Models/Sprint.js
scrumChores.models.sprint = (function (SprintID, SpritnName, SprintStartDate, SprintEndDate) {
    var self = this;
    self.SprintID = ko.observable(SprintID);
    self.SprintName = ko.observable(SpritnName);
    self.SprintStartDate = ko.observable(SprintStartDate);
    self.SprintEndDate = ko.observable(SprintEndDate);
    self.SprintDateRange = ko.computed(function () {
        return this.SprintStartDate() + ' - ' + this.SprintEndDate();
    }, self);
});
         